devenv/dev-dashboards/panel-timeseries/timeseries-stacking2.json

```diff
@@ -424,7 +424,7 @@
           "stringInput": "-50,-50,-50,-50,-50,-50,-50,-50"
         },
         {
-          "alias": "-100",
+          "alias": "__expr__",
```

Comment:
These changes don't look right  - cauality of find-and-replace?

Reply:
I wanted to change it because of this https://github.com/grafana/grafana/blob/main/public/app/features/expressions/ExpressionDatasource.ts#L63-L67
Then failing tests led me to change all of them.

Reply:
But I think this is not necessary. Since it is not related to the expressions at all.

Reply:
I think in _this_ file, this is just an alias describing the metric values of `[-100,-100,-100,-100]` and not the magic value for expressions.

Reply:
you are right. Nice catch. Reverted that part.